x-pack/legacy/plugins/lens/public/xy_visualization/xy_expression.test.tsx

```diff
@@ -441,6 +580,31 @@ describe('xy_expression', () => {
       expect(component.find(Settings).prop('rotation')).toEqual(90);
     });
 
+    test('onBrushEnd returns correct context data', () => {
+      const { args } = sampleArgs();
+
+      const wrapper = mountWithIntl(
+        <XYChart
+          data={dateHistogramData}
+          args={{
+            ...args,
+            layers: [dateHistogramLayer],
+          }}
+          formatFactory={getFormatSpy}
+          timeZone="UTC"
+          chartTheme={{}}
+          executeTriggerActions={executeTriggerActions}
+        />
+      );
+
+      wrapper
+        .find(Settings)
+        .first()
+        .prop('onBrushEnd')!(1585757732783, 1585758880838);
+
+      expect(executeTriggerActions.mock.calls[0]).toMatchSnapshot();
```

Comment:
The test says "returns correct context data", but I don't think snapshot testing is very good at guaranteeing correctness. Can you assert something more specific, such as:

```
.toHaveBeenCalledWith(expect.objectContaining({
  range: [1000, 2000],
});
```